extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -201,6 +209,36 @@ export class SubscriptionService {
     return this.dialog_;
   }
 
+  /**
+   * Select and activate all platforms. Do nothing if the viewer can authorize the user.
+   * @return {!Promise}
+   */
+  selectAndActivatePlatform() {
+    return this.initialize_().then(() => {
+      if (this.doesViewerProvideAuth_ || this.platformConfig_['alwaysGrant']) {
+        return;
+      }
+
+      return this.selectAndActivatePlatform_();
+    });
+  }
+
+  /**
+   * @return {!Promise<boolean>}
+   */
+  getGrantStatus() {
```

Comment:
You're not using this anywhere, do you have a design approved where you 100% know that you'll need this?